.github/workflows/build.yml

```diff
@@ -412,6 +412,14 @@ jobs:
           workdir: control-plane
           tags: |
             public.ecr.aws/hashicorp/${{ env.repo }}-control-plane-fips:${{ env.version }}-ubi
+            docker.io/hashicorp/${{ env.repo }}-control-plane-fips:${{ env.version }}-ubi
           redhat_tag: quay.io/redhat-isv-containers/6486b1beabfc4e51588c0416:${{env.version}}-ubi # this is different than the non-FIPS one
           extra_build_args: |
             GOLANG_VERSION=${{ needs.get-go-version.outputs.go-version }}
+tags: |
+            docker.io/hashicorp/${{ env.repo }}-control-plane-fips:${{ env.version }}-ubi
+          dev_tags: |
```

Comment:
Lgtm - but is there a spacing issue? Unsure why the editor isn't picking up `dev_tags` as an argument to pass in

Reply:
oh good catch let me fix that!